files/ui/graphicspage.ui

```diff
@@ -124,7 +124,7 @@
         <item row="1" column="1">
          <widget class="QRadioButton" name="customRadioButton">
           <property name="text">
-           <string>Custom:</string>
+           <string>Custo&amp;m:</string>
```

Comment:
This doesn't look right. Did we get an extra character added when we didn't mean to?

Reply:
No idea how that happened.